docs/components.adoc

```diff
@@ -0,0 +1,173 @@
+* Statistics will not be up to date and the alerts and errors shown in the admin portal will not be triggered
+* Because accounting will not work, authorizations will not be correct
+* Jobs will not be processed and will start accumulating in `backend-redis`
+
+== backend-cron
+
+=== Responsibilities
+* This is a simple task that acts as a cron scheduler to requeue failed jobs
+* When a `backend-worker` job fails, it is pushed to a "failed jobs" queue so that it can be retried later. Jobs can fail, for example, when there's a Redis timeout
+* It is also responsible for deleting the stats of services that have been removed. This is run every 24h
+
+=== Impact when down
+* Failed jobs will not be rescheduled
+* If crashes in the middle of the delete process, it will just continue the next time it runs
+* If 3scale deployment is working correctly, the failed jobs queues will be empty almost at all times, so `backend-cron` being down is not critical
+
+== backend-redis
+
+=== Responsibilities
+* It is the DB used by `backend-listener` and `backend-worker`
+* It is used to both data persistance (metrics...) and enqueued jobs
+
+=== Impact when down
+* `backend-listener` and `backend-worker` stop working
+
+== system-app
+
+=== Responsibilities
+* Developer and Admin Portal UI/API
+* 3scale APIs (Accounts, Analytics)
+
+=== Impact when down
+* Developer and Admin Portal UI/API will be down
+* 3scale APIs (Accounts, Analytics) will be down
+* `apicast` will not be able to retrieve the gateway configuration, so new `apicast` deployments will not work
+* But current `apicast` pods will still serve traffic with latest downloaded configuration
+
+== system-sidekiq
+
+=== Responsibilities
+* It is the job manager used by `system-app` to process jobs in the background asynchronously
+
+=== Impact when down
+* Emails are not sent
+* Communication with `backend-listener` breaks: changes in Admin Portal will not propagate to Backend
+* Backend alerts will not be triggered
+* Webhooks will not be triggered
+* Zync will not receive any updates
+* Background jobs will not be processed and will start accumulating in `system-redis`
+
+== system-mysql
+
+=== Responsibilities
+* It is the DB used by `system-app`
+
+=== Impact when down
+* `system-app` stops working
+
+== system-redis
+
+=== Responsibilities
+* It is the DB used by `system-app` as a queue for sidekiq jobs processed by `system-sidekiq`
+
+=== Impact when down
+* Background jobs are not stored
+* Billing does not run
+
+== system-memcache
+
+=== Responsibilities
+* Memcached is an ephemeral cache of values used to speed-up the performance of the `system-app` web application
+
+=== Impact when down
+* `system-app` misses its cache (mostly used by the UI to speed-up page loading but also by billing), making `system-app` to run slightly slower while the cache is rebuilt
```

Comment:
misses? Sounds like `system-app` is sasd because it lost it's little memcached ... :smile: 

```suggestion
* `system-app` will run slightly slower (UI page loading times will be worse) while the cache is not accessible. Cache will be rebuilt once the memcached instance is back online.
```